airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/abstract_file_based_stream.py

```diff
@@ -70,8 +70,7 @@ def list_files(self) -> List[RemoteFile]:
         List all files that belong to the stream.
 
         The output of this method is cached so we don't need to list the files more than once.
-        This means we won't pick up changes to the files during a sync. This meethod uses the
-        get_files method which is implemented by the concrete stream class.
+        This means we won't pick up changes to the files during a sync. This method uses the
```

Comment:
nit: this sentence ends abruptly, did we delete the last line of the docstring by accident?

Reply:
updated